jython/com.ibm.wala.cast.python.jython3/source/com/ibm/wala/cast/python/parser/PythonParser.java

```diff
@@ -2446,7 +2446,9 @@ public String getSignature() {
               return "script " + scriptRelativePath.toString();
             }
           }
-          return null; // Not found.
+
+          // not found, so revert to name
+          return this.getName();
```

Comment:
I'll need to investigate this one.

Reply:
the logic is that we return name above if there is no python path, so that seemed to make sense here.

Reply:
That makes sense. I think we wrote this to be backwards compatible so that if a path isn't specified, it behaves as it did before. But maybe we did that by specifying a default path if one wasn't given.